### PR TITLE
Update code intel team's 2020 Q1 OKR progress

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

File: company/okrs/2020_q1.md [hunk 1]
```diff
@@ -57,12 +57,16 @@
 1. **LSIF is adopted by customers**
    1. At least three customers have > 50% of code intel queries powered by LSIF (excluding fallback references)
```
Comment (nicksnyder) [inline company/okrs/2020_q1.md]: What is the status on these first two KRs?

Comment (creachadair) [reply company/okrs/2020_q1.md]: As far as I know we have not hit either (1) or (2) here.

Comment (nicksnyder) [reply company/okrs/2020_q1.md]: We shouldn't remove KRs, but I do want there to be an update here. The update can be "no progress, we are pursuing this by making LSIF work on open source (LINK TO RFC)" or whatever is accurate

Comment (creachadair) [reply company/okrs/2020_q1.md]: I added forward references to the notes I added below. Would you like to see more detail than this?

**Pull request merged**